### DISCUSSION TITLE
openwrt-k批量添加网口

### DISCUSSION THREAD

**Discussion by FawnACO** [chenmozhijin/OpenWrt-K#40] [General]
x86设备基本都是多网口设备，但是没有批量快捷的增加网口的方式，你这只能一个一个手动增加

**Comment by chenmozhijin** [comment]
我可能没理解你的意思，一般来说如果要添加网口到网桥的话直接打勾不就行了么？
![截图](https://github.com/chenmozhijin/OpenWrt-K/assets/110812055/8d0b0b54-0ecf-4b5e-80b2-a7e4f20581ca)
你是希望写一个脚本将所有未使用的以太网适配器添加到指定网桥吗？

**Comment by FawnACO** [comment]
恩，是我找错位置了，大部分openwrt网口是自动分配wan,lan，如果是更换直接在接口-lan-编辑里选择，昨天配置的时候没看设备里。尴尬了。
